starboard/nplb/system_get_total_cpu_memory_test.cc

```diff
@@ -26,9 +26,6 @@ TEST(SbSystemGetTotalCPUMemoryTest, SunnyDay) {
 }
 
 TEST(SbSystemGetTotalCPUMemoryTest, PrintValues) {
-  int64_t bytes_reserved = SbSystemGetTotalCPUMemory();
```

Comment:
tiny nit, but for debugging it's better to have the variable call and print statement on separate lines. I'd just plug the variables into the stringstream statements below.